### PR TITLE
fixes raw hex field

### PR DESCRIPTION
**Pull request by skuhlmann**

### CODE REVIEW

File: src/formBuilder/formBuilder.jsx [hunk 1]
```diff
@@ -80,7 +80,12 @@ const FormBuilder = props => {
   };
 
   const buildABIOptions = (abiString, serialTag = false) => {
-    if (!abiString || typeof abiString !== 'string') return;
+    if (
+      !abiString ||
+      typeof abiString !== 'string' ||
+      abiString.slice(0, 2) === '0x'
```
Comment (jordanlesich) [inline src/formBuilder/formBuilder.jsx]: not necessary to implement, but we also have the validate.address() utilities for stuff like this. Might be helpful.

File: src/utils/txHelpers.js [hunk 2]
```diff
@@ -158,10 +164,13 @@ const argBuilderCallback = Object.freeze({
     ];
   },
   proposeActionSafe({ values, formData }) {
-    const hexData = safeEncodeHexFunction(
-      JSON.parse(values.abiInput),
-      values?.abiArgs || [],
-    );
+    const hexData =
```
Comment (jordanlesich) [inline src/utils/txHelpers.js]: It all checks out. The only thing I'd say is that we're doing the same thing three times over. Can we refactor this into a function? Something with a nice semantic name like ensureHex() or whatever?

File: src/formBuilder/formBuilder.jsx (hunk 1 above)
Comment (jordanlesich) [reply src/formBuilder/formBuilder.jsx]: Might be good to have a validate.hex() if such a thing is possible.

Comment (santteegt) [reply src/formBuilder/formBuilder.jsx]: there are `web3.utils.isHexStrict` or `ethers.utils.isHexString`

**Comment by skuhlmann** [comment]
@santteegt @jordanlesich 

thanks for the feedback! i've addressed these and will get this merged.

**Pull request merged**